app/views/shared/_footer.html.erb

```diff
@@ -1,23 +1,8 @@
-<footer class="bg-light footer">
-  <div class="container">
-    <div class="row">
-      <div class="col-lg-6 text-center text-lg-start my-auto h-100">
-        <ul class="list-inline mb-2">
-          <li class="list-inline-item"><a href="#">About</a></li>
-            <li class="list-inline-item"><span>⋅</span></li>
-            <li class="list-inline-item"><a href="#">Contact</a></li>
-            <li class="list-inline-item"><span>⋅</span></li>
-        </ul>
-          <p class="text-muted small mb-4 mb-lg-0">This website was coded with <i class="fas fa-heart"></i> at Le Wagon.</p>
-      </div>
-        <div class="col-lg-6 text-center text-lg-end my-auto h-100">
-          <ul class="list-inline mb-0">
-            <li class="list-inline-item"><a href="#"><i class="fab fa-lg fa-facebook-f"></i></a></li>
-            <li class="list-inline-item"><a href="#"><i class="fab fa-lg fa-github"></i></a></li>
-            <li class="list-inline-item"><a href="#"><i class="fab fa-lg fa-twitter"></i></a></li>
-            <li class="list-inline-item"><a href="#"><i class="fab fa-lg fa-instagram"></i></a></li>
-          </ul>
-        </div>
-      </div>
-    </div>
-</footer>
+<div class="footer">
```

Comment:
It would be better to use a footer tag instead. It's just like a regular div, but it's semantic, making the site more accessible and the code easier to be read :smiley: